canonicalwebteam/discourse/parsers/base_parser.py

```diff
@@ -286,18 +300,48 @@ def _parse_metadata(self, index_soup, section_name):
                     if value.find("a"):
                         row_dict["topic_name"] = value.find("a").text
 
-                    # Beautiful soup renders URLs as anchors
-                    # Avoid that default behaviour
-                    if value.find("a") and (
-                        value.find("a")["href"] == value.find("a").text
-                    ):
-                        value.contents[0] = value.find("a").text
+                        # Only engage pages need a link
+                        if value.findAll("a", href=True):
+                            if value.find("a")["href"] == value.find("a").text:
+                                value.contents[0] = value.find("a").text
+
+                        else:
+                            error = MissingLinkError(
+                                f"Warning: Link not found when parsing row {index} \"{row_dict['topic_name']}\"\
+                                {titles[index]}. \
+                                This Engage page has been skipped."
+                            )
+                            flask.current_app.extensions[
+                                "sentry"
+                            ].captureMessage(error)
```

Comment:
This is no longer a message (string), it is a `MissingLinkError` exception object. I don't suppose `captureMessage` is going to work.